src/block/block_tiered.c

```diff
@@ -0,0 +1,71 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+/*
+ * __wt_block_tiered_flush --
+ *     Flush the tiered for this file.
```

Comment:
`the tiered` seems to be missing a noun. `the tiered data`? `the tiered object file`?